resbaz/resbazTucson2025.qmd

```diff
@@ -0,0 +1,111 @@
+
+- **Birds-of-a-Feather Sessions:** Join or lead informal discussion groups centered around specific research interests, digital tools, or areas of expertise. These sessions foster organic conversations and networking opportunities among like-minded attendees.
+
+- **Lightning Talks:** Attend or present quick, five-minute presentations showcasing a variety of projects, research findings, or insights on digital tools. Lightning talks offer a rapid-fire way to expose yourself to numerous ideas and topics.
+
+The Unconference Day is all about participation, collaboration, and sharing knowledge. Attendees can propose and lead sessions, contribute to discussions, or simply engage as active learners. The schedule will be dynamic, allowing you to join sessions that pique your curiosity and align with your interests.
+
+Don't miss this opportunity to shape your own ResBaz experience and connect with fellow researchers in an interactive and engaging environment!
+
+## Hacky Hour
+
+**4:00 PM, [Snakes and Lattes](https://g.co/kgs/fmn8QX1), 988 E University Blvd, Tucson, AZ 85719**
+
+Join us after the unconferece for our [regular weekly thursday Hacky Hour meetup](/#hacky-hour).
+
+## Sponsors
+
+```{=html}
+<div class="sponsor-logos">
+  <a class="block-a" href="https://www.arizona.edu" title="University of Arizona">
+    <img src="/img/logos/block_a.svg"/>
+  </a>
+
+  <div class="ua-groups">
+    <a href="https://lib.arizona.edu" title="University Libraries">
+      <img src="/img/logos/ua_libraries.svg" style="height:1.25em"/>
+    </a>
+    <a href="https://datainsight.arizona.edu" title="Institute for Computing and Data-Enabled Insight">
+      <img src="/img/logos/ua_icdi.svg" style="height:2em"/>
+    </a>
+    <a href="https://datascience.arizona.edu" title="Data Science Institute">
+      <img src="/img/logos/ua_dsi.svg" style="height:1.75em"/>
+    </a>
+  </div>
+</div>
+```
+
+
+## Accessibility
+
+We are committed to making this workshop accessible to everybody. For any accommodation requests, please contact us at <ResBazAZ+JEDI@gmail.com>. We will do our best to make the event inclusive to all.
+
+## Previous years
+
+- [ResBaz Tucson 2024](/resbaz/Arizona2024.qmd)
+- [ResBaz Arizona 2023](/resbaz/Arizona2023.qmd)
+- [ResBaz Tucson 2022](/resbaz/resbazTucson2022.qmd)
+- [ResBaz Tucson 2021](/resbaz/resbazTucson2021.qmd)
+- [ResBaz Tucson 2020](/resbaz/resbazTucson2020.qmd)
+- [ResBaz Tucson 2019](/resbaz/resbazTucson2019.qmd)
+- [ResBaz Tucson 2018](/resbaz/resbazTucson2018.qmd)
+
+## Comments
+
+<div id="comment-section"></div>
+
+<script>
+initComments({
+  node: document.getElementById("comment-section"),
+  defaultHomeserverUrl: "https://matrix.cactus.chat:8448",
+  serverName: "cactus.chat",
+  siteName: "resbazaz",
+  commentSectionId: "festival2025",

```

Comment:
I'm guessing this is a @julianpistorius job to set this up?

Reply:
This 'should just work' :tm: :sweat_smile: :crossed_fingers: 

- https://cactus.chat/docs/getting-started/quick-start/
- https://cactus.chat/docs/client/introduction/